.travis.yml

```diff
@@ -22,12 +22,12 @@ before_install:
   - tar zxf thrift-0.7.0.tar.gz
   - cd thrift-0.7.0
   - chmod +x ./configure
-  - ./configure --disable-gen-erl --disable-gen-hs --without-php --without-nodejs --without-ruby --without-haskell --without-erlang
+  - ./configure --disable-gen-erl --disable-gen-hs --without-ruby --without-haskell --without-erlang --without-php
   - sudo make install
   - cd ..
 
 env:
-  - HADOOP_PROFILE=hadoop-1 TEST_CODECS=uncompressed
+  - HADOOP_PROFILE=default TEST_CODECS=uncompressed,brotli
```

Comment:
While I don't have any objections to removing the hadoop-1 profile, please note that the hadoop-1 travis builds don't fail any more, so it's not strictly necessary.

Reply:
As long as we are in agreement that we don't need it any more, then I'm for removing it from testing.